libs/client/conference/connect/data-access/src/lib/state/selectors/connect.selectors.ts

```diff
@@ -28,3 +28,7 @@ export const selectCandidates = createSelector(
 export const selectCurrentCandidate = createSelector(selectCandidates, (candidates) =>
   candidates.find((candidate) => candidate.candidateType === CandidateType.UNDECIDED)
 );
+
+export const selectNoCandidates = createSelector(selectCandidates, (candidates) =>
+  Boolean(candidates.length)
```

Comment:
Can we switch this around so that this is set to `true` when there are no candidates? Because right now, it'll be `false` when there's no candidates, right? It feels a little backwards to me if that's correct (I could totally be wrong though lol). But I'm not sure if using `!` works well in selectors?